src/components/AdminNavBar/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+import { AppBar, Typography } from "@mui/material";
+import { routes } from "../../constants/routes";
+import AccountAvatar from "../NavBar/AccountAvatar";
+import logo from "../../img/pd-logo.svg";
+
+const AdminNavBar = () => {
```

Comment:
Great, this looks better now. Can you also think about reducing code duplication between AdminNavBar and NavBar, where possible? The clickable logo section looks like a good candidate. Maybe call it HomeButton and reuse it in AdminNavBar and NavBar. 

AccountAvatar is reused between these two components, which is good, but its files are positioned in NavBar folder.  Can you propose a different file organization so it reflects this component is shared?